provisioner/shell/provisioner_test.go

```diff
@@ -491,14 +491,14 @@ func TestProvisioner_RemoteFileDefaultsToScriptnnnn(t *testing.T) {
 		t.Fatalf("should not have error: %s", err)
 	}
 
-	remoteFileRegex := regexp.MustCompile("script_[0-9]{4}.sh")
+	remoteFileRegex := regexp.MustCompile("script_[0-9]{1,4}.sh")
```

Comment:
This is to fix the following:
```
--- FAIL: TestProvisioner_RemoteFileDefaultsToScriptnnnn (0.00s)
    provisioner_test.go:497: remote_file did not default to script_nnnn.sh: "script_6.sh"
FAIL
```

Reply:
I think I've seen this test failing elsewhere -- think you could make a separate PR for it so I can rebase the currently failing branches?